config_center/nacos/README.md

```diff
@@ -0,0 +1,68 @@
+# Dubbo-go Config-Center Sample
+
+## 1. Introduction
+
+This example shows dubbo-go's dynamic configuration feature with Nacos as config-center.
+
+## 2. How to run
+
+### Add configuration to the configuration center of nacos
+
+```go
+dynamicConfig, err := config.NewConfigCenterConfigBuilder().
+SetProtocol("nacos").
+SetAddress("127.0.0.1:8848").
+Build().GetDynamicConfiguration()
+
+if err != nil {
+    panic(err)
+}
+
+if err := dynamicConfig.PublishConfig("dubbo-go-samples-configcenter-nacos-server", "dubbo", configCenterNacosServerConfig); err != nil {
+    panic(err)
+}
+time.Sleep(time.Second * 10)
+```
+Open `https://localhost:8848/nacos/` with browser, make sure the relevant configuration is already in place in nacos.
```

Comment:
I think just introduce the configuration in the config center, and how to use it.